external-builds/pytorch/build_prod_wheels.py

```diff
@@ -169,12 +169,15 @@ def get_installed_package_version(dist_package_name: str) -> str:
     lines = capture(
         [sys.executable, "-m", "pip", "show", dist_package_name], cwd=Path.cwd()
     ).splitlines()
+    if not lines:
+        raise ValueError(f"Did not find installed package '{dist_package_name}'")
     prefix = "Version: "
     for line in lines:
         if line.startswith(prefix):
             return line[len(prefix) :]
+    joined_lines = "\n".join(lines)
     raise ValueError(
-        f"Did not find installed package {dist_package_name} in {'\n'.join(lines)}"
+        f"Did not find Version for installed package '{dist_package_name}' in output:\n{joined_lines}"
     )
```

Comment:
Sample output of this failure if `Version: ` is somehow not in the `pip show` output:
```
ValueError: Did not find Version for installed package 'numpy' in output:
Name: numpy
Version: 2.3.1
Summary: Fundamental package for array computing in Python
Home-page: https://numpy.org
Author: Travis E. Oliphant et al.
Author-email: 
License: Copyright (c) 2005-2025, NumPy Developers.
 All rights reserved.

 Redistribution and use in source and binary forms, with or without
 modification, are permitted provided that the following conditions are
 met:

... (continues to print the full license text)
```